tests/Avalonia.Layout.UnitTests/FullLayoutTests.cs

```diff
@@ -113,17 +113,17 @@ public void Test_ScrollViewer_With_TextBlock()
                 Assert.Single(presenters);
 
                 var presenter = presenters[0];
-                Assert.Equal(new Size(190, 190), presenter.Bounds.Size);
+                Assert.Equal(new Size(196, 196), presenter.Bounds.Size);
```

Comment:
Why was this `190` instead of the `196`?

Reply:
I have changed grid column and row size from fixed to auto in xaml template.

Reply:
That is the result of adding `ScrollBarWidth` and `ScrollBarHeight` to theme accent, the theme accent is not applied int the test so size is different. You can add to the test: Test_ScrollViewer_With_TextBlock
```C#
                window.Resources["ScrollBarWidth"] = 10.0;
                window.Resources["ScrollBarHeight"] = 10.0;

                window.Show();
                window.LayoutManager.ExecuteInitialLayoutPass(window);
```